app/SectionApp/TodoContainer/Models/Step.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\SectionApp\TodoContainer\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * Class Step
+ *
+ * This class represents a step in the application.
+ */
+class Step extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<int, string>
+     */
+    protected $fillable = [
+        'todo_id',
+        'description',
+        'order',
+    ];
+
+    /**
+     * Get the post that owns the comment.
```

Comment:
Check lại doc comment class/function khác luôn nhé
```suggestion 
     * Get the todo that owns the steps.
```